demo/src/javascripts/components/demos/arc.js

```diff
@@ -108,16 +111,14 @@ export default class ArcDemo extends Component {
     selectedFeature = selectedFeature || features[362];
 
     const {flows, centroid, name} = selectedFeature.properties;
-    const arcs = [];
-
-    for (let toId in flows) {
+    const arcs = Object.keys(flows).reduce((acc, toId) => {
       const f = features[toId];
-      arcs.push({
+      return [...acc, {
```

Comment:
Although I agree with you both, we have to keep in mind this is only a demo 😃 

I'll make the change back, but this kinda makes me think about the diff I did to remove the `switch` statement and replace it by a hashmap. Although less elegant, the solution had the advantage to be `O(1)`.

My question being what's the line where performance becomes more important than code prettiness?